jekyll/_cci2/v.2.19-overview.adoc

```diff
@@ -9,16 +9,30 @@ This document provides a summary of features and product notes for the release o
 
 ## Requirements for Upgrading
 
+WARNING: Before upgrading to 2.19.3, if you are using an IAM role scoped to a non-root path, you will need to unset the `OUTPUT_PROCESSOR_USE_NAIVE_ROLE_MAPPING` environment variable in your output processor customization script. See the https://circleci.com/docs/2.0/customizations/#service-configuration-overrides[Customizations Guide] in our documentation for more information on using customization scripts.
```

Comment:
Could we include some examples here so customers can more easily tell if this affects them/

Reply:
Yes I will ask around for some examples and stick them in later :-)